packages/dappmanager/src/modules/installer/runPackages.ts

```diff
@@ -76,3 +82,24 @@ export async function runPackages(
     log(pkg.dnpName, "Package started");
   }
 }
+
+/**
+ * Check if a package contains compose feature pid
+ * in any of its services
+ */
+export function hasPid(pkg: InstallPackageData): boolean {
```

Comment:
Can you move this to a utility function in compose? Rename to `composeHasPid()`. Also you can simplify with

```ts
for (const service of Object.values(pkg.compose.services) {
  if (service.pid) return true
}
return false
```

Reply:
done, double check since there is a change in the PR logic